muxpi/set-up-image.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+set -e
+
+#If you want to run the neptune image, run `./set-up-image.sh neptune`
+NEPTUNE_IMAGE="$1"
+COUNTER=5
```

Comment:
Why is the counter defined here, but the C variable is defined somewhere else? Is Counter needed for something else before that? I think it should be moved, if it can. It has nothing to do with the image so doesn't need to be grouped with this variable.

Reply:
Defined it just on top of the while loop now.